ha/const.py

```diff
@@ -168,6 +170,7 @@
 RESOURCE_TYPE = "resource_type"
 IEM_DESCRIPTION="WS0080010001, Node, The cluster has lost $host server. System is running in degraded mode. " \
                 "For more information refer the Troubleshooting guide. Extra Info: host=$host; status=$status;"
+logger_utility_iec_cmd="logger -i -p local3.err"
```

Comment:
All constants specific to alerts are in ha/const.py  
Please see if this can be moved there